packages/babel-plugin-transform-es2015-parameters/test/fixtures/parameters/default-rest/actual.js

```diff
@@ -0,0 +1,15 @@
+const a = 1;
+function rest(b = a, ...a) {
```

Comment:
Oh didn't realize that our discussion is kind off offtopic. I can't really say anything about this PR would have to dive deeper first.